src/components/RangeSlider/components/SingleThumb/SingleThumb.scss

```diff
@@ -29,6 +25,12 @@ $range-thumb-shadow-focus: 0 0 0 rem(1px) color('indigo');
   align-items: center;
   flex: 1 1 auto;
   height: $range-thumb-size;
+
+  input {
+    padding: rem(12px) 0;
```

Comment:
In order to make the heights consistent, I need to use base * 3 for padding, which doesn't exist in the Sass spacing map.